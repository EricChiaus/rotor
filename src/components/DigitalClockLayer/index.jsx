import React, { useContext } from 'react';
import { TimeContext } from '../../contexts/TimeContext';
import DigitalClock from './../DigitalClock';

const DigitalClockLayer = () => {
    const { hours, minutes, seconds } = useContext(TimeContext);
    return <DigitalClock hours={hours} minutes={minutes} seconds={seconds} />;
};

export default DigitalClockLayer;
