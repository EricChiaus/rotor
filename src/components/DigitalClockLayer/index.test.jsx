import { render } from '@testing-library/react';
import DigitalClockLayer from '.';

describe('<DigitalClockLayer />', () => {
    it('matches the snapshot', () => {
        const { container } = render(<DigitalClockLayer />);
        expect(container).toMatchSnapshot();
    });
});
