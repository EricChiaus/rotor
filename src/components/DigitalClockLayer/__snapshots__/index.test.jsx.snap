// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DigitalClockLayer /> matches the snapshot 1`] = `
<div>
  <div
    class="digital-clock"
  >
    <div
      class="zero digit"
    >
      <span
        class="d1"
      />
      <span
        class="d2"
      />
      <span
        class="d3"
      />
      <span
        class="d4"
      />
      <span
        class="d5"
      />
      <span
        class="d6"
      />
      <span
        class="d7"
      />
    </div>
    <div
      class="zero digit"
    >
      <span
        class="d1"
      />
      <span
        class="d2"
      />
      <span
        class="d3"
      />
      <span
        class="d4"
      />
      <span
        class="d5"
      />
      <span
        class="d6"
      />
      <span
        class="d7"
      />
    </div>
    <div
      class="dots"
    />
    <div
      class="zero digit"
    >
      <span
        class="d1"
      />
      <span
        class="d2"
      />
      <span
        class="d3"
      />
      <span
        class="d4"
      />
      <span
        class="d5"
      />
      <span
        class="d6"
      />
      <span
        class="d7"
      />
    </div>
    <div
      class="zero digit"
    >
      <span
        class="d1"
      />
      <span
        class="d2"
      />
      <span
        class="d3"
      />
      <span
        class="d4"
      />
      <span
        class="d5"
      />
      <span
        class="d6"
      />
      <span
        class="d7"
      />
    </div>
    <div
      class="dots"
    />
    <div
      class="zero digit"
    >
      <span
        class="d1"
      />
      <span
        class="d2"
      />
      <span
        class="d3"
      />
      <span
        class="d4"
      />
      <span
        class="d5"
      />
      <span
        class="d6"
      />
      <span
        class="d7"
      />
    </div>
    <div
      class="zero digit"
    >
      <span
        class="d1"
      />
      <span
        class="d2"
      />
      <span
        class="d3"
      />
      <span
        class="d4"
      />
      <span
        class="d5"
      />
      <span
        class="d6"
      />
      <span
        class="d7"
      />
    </div>
  </div>
</div>
`;
